helm/dagster/Chart.yaml

```diff
@@ -30,8 +30,8 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     condition: redis.internal
 maintainers:
-  - name: Rex Ledesma
-    email: rex@elementl.com
-    url: https://dagster.io
-icon: https://dagster.io/images/logo.png
+  - name: Elementl
+    email: support@elementl.com
```

Comment:
Is this a real email address?

Reply:
And even if it is, do we want to perhaps have a support@dagster.io too/instead?

Reply:
it's a real email address. it was easy to create this since we own the google workspace domain. Not sure about making an dagster.io one though.